main/ZgatewayPilight.ino

```diff
@@ -186,7 +186,7 @@ void MQTTtoPilight(char* topicOri, JsonObject& Pilightdata) {
     }
 #  ifdef Pilight_rawEnabled
     if (Pilightdata.containsKey("rawEnabled")) {
-      Log.notice(F("Setting PiLight raw output enabled: %s" CR), Pilightdata["rawEnabled"]);
+      Log.notice(F("Setting PiLight raw output enabled: %s" CR), Pilightdata["rawEnabled"] ? "true" : "false");
```

Comment:
Thanks, alternatively we could use %T instead of %s https://github.com/thijse/Arduino-Log#log-events